app.js

```diff
@@ -0,0 +1,13 @@
+const joinNowButton = document.getElementById("join-now-button");
+
+joinNowButton.addEventListener("click", alertBox);
+
+function alertBox(event) {
+  event.preventDefault();
+  const inputElement = document.getElementById("email").value;
+  return inputElement != ""
+    ? alert(
+        ` “Thank You! Your email ${inputElement} *successfully added to our subscribers list.*”`
+      )
+    : alert("“Please enter your email.”");
```

Comment:
toks ligas returnas gana sudėtingai skaitosi. Gal toks lengviau perskaitomas?
```
const message = inputElement !== "" ? "Thank you! Your email ........ " : "Please enter ....";
return alert(message)
```

Reply:
arba `return alert(inputElement !== "" ? "Thank you! Your email ........ " : "Please enter ....")`